python/GafferOFXUI/OFXImageNodeUI.py

```
@@ -0,0 +1,62 @@
+	plugs = {
+
+		"pluginId" : [
+
+			"description",
+			"""
+			The OFX id of the plugin to load.
+			""",
+
+		],
+
+	}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)